rpcserverhelp.go

```diff
@@ -99,6 +99,14 @@ var helpDescsEnUS = map[string]string{
 	"decodescript--synopsis": "Returns a JSON object with information about the provided hex-encoded script.",
 	"decodescript-hexscript": "Hex-encoded script",
 
+	// GenerateCmd help
+	"generate--synopsis": "Generates a set number of blocks (simnet or regtest only) and returns a JSON\n" +
+		" array of their hashes.",
+	"generate-numblocks": "Number of blocks to generate",
+
+	// GenerateResult help
+	"generateresult-blockhashes": "The hashes, in order, of blocks generated by the call",
```

Comment:
This would become:

`"generateresult--result0": "The hashes, in order, of blocks generated by the call"`,